src/components/MetadataCollections/MetadataCollections.js

```diff
@@ -192,15 +203,53 @@ class MetadataCollections extends React.Component {
     />
   );
 
+  cacheFilter(activeFilters, searchValue) {
+    localStorage.setItem('fincConfigCollectionFilters', JSON.stringify(activeFilters));
+    localStorage.setItem('fincConfigCollectionSearchString', JSON.stringify(searchValue));
+  }
+
+  resetAll(getFilterHandlers, getSearchHandlers) {
+    localStorage.removeItem('fincConfigCollectionFilters');
+    localStorage.removeItem('fincConfigCollectionSearchString');
+
+    // reset the filter state to default filters
+    getFilterHandlers.state(defaultFilter.state);
+
+    // reset the search query
+    getSearchHandlers.state(defaultSearchString);
+
+    this.setState({
+      storedFilter: defaultFilter,
+      storedSearchString: defaultSearchString,
+    });
+
+    return (this.props.history.push(`${urls.collections()}?filters=${defaultFilter.string}`));
+  }
+
+  handleClearSearch(getSearchHandlers, onSubmitSearch, searchValue) {
+    localStorage.removeItem('fincConfigCollectionSearchString');
+    localStorage.removeItem('fincConfigCollectionSearchIndex');
+
+    searchValue.query = '';
+
+    getSearchHandlers.state({
+      query: '',
+    });
+
+    // TODO: url still contains old qindex?!
+  }
+
   render() {
     const { intl, queryGetter, querySetter, onSelectRow, selectedRecordId, collection } = this.props;
     const count = collection ? collection.totalCount() : 0;
 
     return (
       <div data-test-collections>
         <SearchAndSortQuery
-          initialFilterState={{ metadataAvailable: ['yes'] }}
-          initialSearchState={{ query: '' }}
+          // initialFilterState={{ metadataAvailable: ['yes'] }}
```

Comment:
Please remove commented out code. Git still remembers this change.

Reply:
code, which is commented out has been deleted